handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -824,23 +825,18 @@ public final boolean isServer() {
      */
     public abstract SSLSessionContext sessionContext();
 
-    /**
-     * Creates a new {@link SslHandler}.
-     *
-     * @return a new {@link SslHandler}
-     */
+    @Deprecated
```

Comment:
IMHO, SslContext shouldn't be a factory for SslHandlers but it has to be to maintain backwards compatibility. The SslHandlerFactory's sslContext() method is need and makes sense if you roll your own implementation but if you're using SslContext then it's strange.